.github/workflows/on_release_notes.yml

```diff
@@ -121,6 +123,12 @@ jobs:
       alias: latest
       detached_mode: true
 
+  publish_layer:
+    needs: release
+    uses: ./.github/workflows/publish_layer.yml
+    with:
+      latest_published_version: ${{ needs.release.outputs.RELEASE_VERSION }}
+
   post_release:
```

Comment:
pr: let's remove `post_release` job. We could have V2 issues being closed after a V1 bugfix release completed. We already track on Projects, so it's best to close any V1 specific issues manually after a release.